### PR TITLE
Implement js package.

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

**Comment by msporny** [comment]
Non-substantive change and @mattcollier is under the Digital Bazaar organization. The W3C script that links him up with this account seems to be busted. Approving and moving on.

File: contexts/credentials-v1.jsonld [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "@context": [{
+    "@version": 1.1
+  }, "https://w3id.org/security/v2", {
+    "cred": "https://w3.org/2018/credentials#",
+    "dc": "http://purl.org/dc/terms/",
+    "sec": "https://w3id.org/security#",
+    "schema": "http://schema.org/",
```
Comment (dlongley) [inline contexts/credentials-v1.jsonld]: Remove "schema"

File: contexts/credentials-v1.jsonld [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "@context": [{
+    "@version": 1.1
+  }, "https://w3id.org/security/v2", {
+    "cred": "https://w3.org/2018/credentials#",
+    "dc": "http://purl.org/dc/terms/",
```
Comment (dlongley) [inline contexts/credentials-v1.jsonld]: Remove "dc"

File: contexts/credentials-v1.jsonld [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "sec": "https://w3id.org/security#",
```
Comment (dlongley) [inline contexts/credentials-v1.jsonld]: Remove "sec"

**Pull request merged**